sys-fs/mdadm/files/mdadm.service

```
@@ -0,0 +1,5 @@
+[Unit]
+Description=Initiates a check run of an MD array's redundancy information.
+
+[Service]
+ExecStart=/usr/sbin/checkarray --cron --all --idle --quiet
```

```suggestion
Type=oneshot
ExecStart=/usr/sbin/checkarray --cron --all --idle --quiet
```
Making it a oneshot service unit also makes sense (cf. , e.g.,`fstrim.service` as shipped in Fedora/Debian)